testing_dir/minesweeper.py

```diff
@@ -36,7 +36,7 @@ def print_stats():
 def update_player_pos(direction):
     x, y = player_pos
     if direction == "up":
-        y -= 1
+        y -= 2  # Error: should be 1
```

Comment:
The decrement for moving "up" is set to 2, which may lead to larger than intended movement. It would be better to decrement by 1 for consistent movement.
```diff
-         y -= 2  # Error: should be 1
+         y -= 1
```
---